internal/checker/api_binding.go

```diff
@@ -10,10 +10,10 @@ import (
 )
 
 // Every cmdApi needs to be attached here in order to be available:
-func attachApi(importLoopData *shared.ImportLoopData, ruleConf *RuleConf, L *lua.LState) {
+func attachApi(importLoopData *shared.ImportLoopData, ruleConf *shared.RuleConfigLayout, L *lua.LState) {
 	apiNamespace := newLuaNamespace()
 
-	apiNamespace.AddField("pkg", getPackageNamespace(L))
+	apiNamespace.AddField("pkg", getPackageNamespace(L, importLoopData))
 	apiNamespace.AddField("sys", getSysInfoNamespace(L))
 	apiNamespace.AddField("fs", getFsNamespace(L, importLoopData))
 	apiNamespace.AddField("info", getInfoNamespace(importLoopData, L))
```

Comment:
Info namespace and Package namespace both takes the same arfuments, but in different order
Please make it consistent (I propose changing Info namespace arguments order)